input/pagecontent/general-requirements.md

```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+
+Must Support elements are treated differently between [AU Core Responder](ActorDefinition-au-core-actor-responder.html) and [AU Core Requester](ActorDefinition-au-core-actor-requester.html) actors, *Must Support* on a profile element **SHALL** be interpreted as follows.
 
 #### AU Core Responder
 An AU Core Responder:
-- **SHALL** be capable of populating all data elements as part of the query results as specified by the [AU Core Server Capability Statement](CapabilityStatement-au-core-server.html).
-- **SHALL** populate all elements labelled *Must Support* unless:
-   - the element is not available for that patient.
-   - a requester does not have access rights to the element (the data is suppressed due to a security or privacy reason), in this case the responder **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data).
-- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element is not available for a patient.
-  - **SHALL NOT** include an element in the resource instance returned as part of query results when the element is optional.
-  - **SHALL** include an element in the resource instance returned as part of query results when the element is mandatory by following the instructions in the section on [Missing data](general-requirements.html#missing-data).
+- **SHALL** correctly populate all _Must Support_ elements if a value is known for the element.
+- **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data) when an element is NOT allowed to be shared.
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is not known.
 
+When a _Must Support_ element requires a more tightly stated obligation, this obligation is specified in the AU Core Responder [obligation extension](https://hl7.org/fhir/extensions/StructureDefinition-obligation.html) on the element definition.
 
 #### AU Core Requester
 An AU Core Requester:
-- **SHALL** be capable of meaningfully processing resources with must support elements
-   - **SHALL** be able to process resource instances containing elements with missing or suppressed information.
-- **MAY** choose to reject non-conformant resources 
-- **SHALL** interpret missing data elements within resource instances as data not present in the source system when querying AU Core Responders
+- **SHALL** accept resources containing any valid value for _Must Support_ elements without error.
+- **SHALL** accept resources containing _Must Support_ elements with [Missing data](#missing-data) or [Suppressed data](#suppressed-data) without error.
```

Comment:
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

Reply:
fixed